src/tracer.cc

```diff
@@ -557,7 +572,7 @@ TraceManager::Trace::StartSpan(
     LOG_TRITONSERVER_ERROR(
         TRITONSERVER_InferenceTraceRequestId(trace, &request_id),
         "getting request id");
-    span->SetAttribute("triton.model_name", model_name);
+    span->SetAttribute("triton.model_name", display_name);
```

Comment:
I was wondering if we need to update the attribute here, as it's `model_name` while it is set with the span name.

Reply:
For request spans span name equals model name

Reply:
Thanks for the clarification!

Reply:
I understand your point, I'll think of something to avoid confusion